src/Microsoft.Crank.Jobs.HttpClient/Program.cs

```diff
@@ -375,7 +375,7 @@ IEnumerable<Task> CreateTasks()
                 Started = startTime.ToLocalTime(),
                 ThroughputBps = workerTasks.Select(x => x.Result.ThroughputBps).Sum(),
                 LatencyMaxMs = Math.Round(workerTasks.Select(x => x.Result.LatencyMaxMs).Max(), 3),
-                LatencyMeanMs = workerTasks.Select(x => x.Result.TotalRequests).Sum() == 0 ? 0 : Math.Round((stopTime - startTime).TotalMilliseconds / workerTasks.Select(x => x.Result.TotalRequests).Sum(), 3),
+                LatencyMeanMs = Math.Round(workerTasks.Select(x => x.Result.LatencyMeanMs).Average(), 3),
```

Comment:
This is assuming that each worker sends the same amount of requests, but we can do that. Meaning that if a worker was sending a single request that is 1ms, and another worker was sending 1000 requests of 1s avg, then it would result in an average of 500ms while it closer to 1s. Whatever, this is closer to reality now.

The only correct option would be to store the TotalLatency and TotalRequests per worker and then do the mean of all these.

Reply:
ah yes, you are correct. In reality we want the sum of TotalLatency of all workers divided by the sum of TotalRequests of all workers.

I can work on this :)